### PR TITLE
Pagination size change

### PR DESCRIPTION
**Pull request by Monsoor-Fayas**
Please check my change

### CODE REVIEW

File: UI/paisa-kanaku/src/components/shared/table/PKTable.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const PKTable = (props) => {
         bordered
         title={props.title}
         pagination={{
-          pageSize: 50,
+          pageSize: 10,
```
Comment (Praga-Dev) [inline UI/paisa-kanaku/src/components/shared/table/PKTable.js:16]: Hi @Monsoor-Fayas 
10 is too low for the pagination, could you **change it to 25 per page**?

**Review by Praga-Dev — CHANGES_REQUESTED** [review]
Please go through the changes which I have mentioned.

File: UI/paisa-kanaku/src/components/shared/table/PKTable.js (hunk 1 above)
Comment (Monsoor-Fayas) [reply UI/paisa-kanaku/src/components/shared/table/PKTable.js:16]: Ok, will change.

Comment (Monsoor-Fayas) [reply UI/paisa-kanaku/src/components/shared/table/PKTable.js]: Changes done

**Review by Praga-Dev — APPROVED** [review]
LGTM 👍

**Pull request merged**